src/detectors/ZDC/ZDC.cc

```diff
@@ -102,5 +106,150 @@ extern "C" {
             app   // TODO: Remove me once fixed
           )
         );
+
+        //side length of hexagonal cells in SiPM-on-tile part of the ZDC
+        auto side_length = 31.3 * dd4hep::mm;
+
+        app->Add(new JOmniFactoryGeneratorT<CalorimeterHitDigi_factory>(
+          "ZDCRawHits", {"HcalFarForwardZDCHits"}, {"ZDCRawHits"},
```

Comment:
Avoid starting collection with Hcal when everything else starts with ZDC.

Reply:
When adding the SiPM-on-tile ZDC to the epic repository (for the geometry), the person reviewing our code (I forget who it was) told us to use this particular name for the hits in this detector.  I can change this back to "ZDCHIts" or "ZDCHcalHits" (which I think sounds more reasonable) if I can get it changed in the epic repository as well, but I think that should be a separate pull request

Reply:
If anything, we should rename all output to HcalFarForwardZDC*, EcalFarFowardZDC*

Reply:
(keep ddh4hep inputs starting with ZDC for now)